brain/openclaw_adapter.py

```
@@ -0,0 +1,183 @@
+        try:
+            async with httpx.AsyncClient(timeout=timeout, proxy=None, trust_env=False) as client:
+                response = await client.post(f"{self.base_url}/neko/send", json=payload)
+                response.raise_for_status()
+                data = response.json()
+        except httpx.TimeoutException:
+            self.last_error = f"OpenClaw request timed out ({self.timeout}s)"
+            return {"success": False, "error": self.last_error}
+        except httpx.HTTPStatusError as exc:
+            self.last_error = f"OpenClaw returned HTTP {exc.response.status_code}"
+            return {"success": False, "error": self.last_error}
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)